backtrace-library/src/androidTest/java/backtraceio/library/breadcrumbs/BacktraceBreadcrumbsTest.java

```diff
@@ -285,7 +285,7 @@ public void testQueueFileShouldNotRollover() {
             // First breadcrumb is configuration breadcrumb, it should be valid
             JSONObject parsedBreadcrumb = new JSONObject(breadcrumbLogFileData.get(0));
             assertEquals("Breadcrumbs configuration", parsedBreadcrumb.get("message"));
-            assertTrue(((int) parsedBreadcrumb.get("id")) == 0);
+            /*assertTrue(((int) parsedBreadcrumb.get("id")) == 0);*/
```

Comment:
Are you sure we want to change breadcrumbs behavior in the event aggregation pull request?

Reply:
+1

Reply:
This is for the breadcrumbs fix you pointed out earlier. Now initial breadcrumb ID won't be 0, it will be `timestamp`. I need to remove these commented out lines, rest of the test still verifies correct/expected behavior as well as end-to-end test in the UI.